app/javascript/packs/copyCode.js

```diff
@@ -0,0 +1,37 @@
+function isClipboardSupported() {
+    return (
+      typeof navigator.clipboard !== 'undefined' && navigator.clipboard !== null
+    );
+  }
+
+  function isNativeAndroidDevice() {
+    return (
+      navigator.userAgent === 'DEV-Native-android' &&
+      typeof AndroidBridge !== 'undefined' &&
+      AndroidBridge !== null
+    );
+  }
+
+function execCopyText() {
+    document.execCommand('copy');
+}
+
+function copyText(event) {
+    const inputValue = event.target.parentNode.nextElementSibling.getElementsByTagName('code')[0].innerText;
+    if (isNativeAndroidDevice()) {
+        AndroidBridge.copyToClipboard(inputValue);
+    } else if (isClipboardSupported()) {
+        navigator.clipboard
+        .writeText(inputValue)
+        .catch(() => {
+            execCopyText();
+        });
+    } else {
+        execCopyText();
+    }
```

Comment:
The alternative when the Runtime PR is merged would be:

```suggestion
    Runtime.copyToClipboard(inputValue);
```

Also all three `isClipboardSupported`, `isNativeAndroidDevice` and `execCopyText` could be removed too.

Reply:
@emgoto. @fdoxyz's PR has been merged.